apdet-hanna/src/apdet.cpp

```diff
@@ -298,6 +298,7 @@ int sumArrElems(int arr[], int size)
 float getMedian(float arr[], int size) {
```

Comment:
You'll probably want to document someplace that this function has the side-effect of sorting the array